cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -468,7 +468,7 @@ <h2 id="toppy" style="background-image:url('');">Do better web tester page</h2>
 <script>window.fetch = () => new Promise(r => setTimeout(r, 90000));</script>
 
 <!-- FAIL(is-on-https): requires a non-localhost http file -->
-<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+<script src="http://0.0.0.0:10503/dobetterweb/third_party/fake-jquery.js"></script>
```

Comment:
nit: why not localhost?

Reply:
Because we explicitly ignore localhost-like urls in the audit. Except we aren't exhaustive so we miss this less common way to specifying localhost :P